Formula/ddns-go.rb

```diff
@@ -0,0 +1,23 @@
+class DdnsGo < Formula
+  desc "Simple and easy-to-use DDNS"
+  homepage "https://github.com/jeessy2/ddns-go"
+  url "https://github.com/jeessy2/ddns-go/archive/refs/tags/v5.3.1.tar.gz"
+  sha256 "afca100b1b2d2b5bdcb0f6d061709136a567978e9cca85b9c0867d9219ead3f5"
+  license "MIT"
+  head "https://github.com/jeessy2/ddns-go.git", branch: "master"
+
+  depends_on "go" => :build
+
+  def install
+    ldflags = %W[
+      -X main.version=v#{version}
+      -X main.buildTime=#{time.iso8601}
+      -extldflags -static -s -w
+    ]
+    system "go", "build", *std_go_args(ldflags: ldflags), "-trimpath"
+  end
+
+  test do
+    system bin/"ddns-go", "-h"
```

Comment:
Can you propose a more substantial test than checking the version or calling for help? These tests rarely ever fail even for broken installations.

>We want tests that don’t require any user input and test the basic functionality of the application. For example `foo build-foo input.foo` is a good test and (despite their widespread use) `foo --version` and `foo --help` are bad tests. 

Ref. https://docs.brew.sh/Formula-Cookbook#add-a-test-to-the-formula

Reply:
Agreed. I have modified the test code to verify if the first line of the output from the `ddns-go` command matches the specified string.